website/content/en/docs/setup/installation/package-managers/helm.md

```diff
@@ -49,32 +53,83 @@ sinks:
 VALUES
 ```
 
-Install Vector:
+### Installing
+
+Once you add the Vector Helm repo, and added a Vector configuration file, install the Vector Agent:
+
+```shell
+helm install vector timberio/vector-agent \
+  --namespace vector \
+  --create-namespace \
+  --values values.yaml
+```
+
+### Updating
+
+Or to update the Vector Agent:
+
+```shell
+helm repo update && \
+helm upgrade vector timberio/vector-agent \
+  --namespace vector \
+  --reuse-values
+```
+
+## Aggregator
+
+The Vector [Aggregator] lets you [transform] your data. For example, dedupe, aggregate, or redact the data before sending it to its final destination.
+
+### Configuring
+
+To check available Helm chart configuration options:
 
 ```shell
-helm install vector vector/vector-agent \
+helm show values timberio/vector-aggregator
+```
+
+This example configuration file lets you use Vector as an Aggregator to parse events to make them human-readable. For more information about configuration options, see the [Configuration] docs page.
+
+```toml
+# Add example
+```
```

Comment:
```suggestion
```yaml
cat <<-'VALUES' > values.yaml
# The Vector Aggregator chart defines a vector
# kubernetes_logs source that is made available to you.
# You do not need to define a log source.
transforms:
  # Adjust as necessary. This remap transform parses a JSON
  # formatted log message, emitting a log if the contents are
  # not valid JSON
  # /docs/reference/transforms/
  remap:
    type: remap
    inputs: ["vector"]
    source: |
      structured, err = parse_json(.message)
      if err != null {
        log("Unable to parse JSON: " + err, level: "error")
      } else {
        . = merge(., structured)
      }
sinks:
  # Adjust as necessary. By default we use the console sink
  # to print all data. This allows you to see Vector working.
  # /docs/reference/sinks/
  stdout:
    type: console
    inputs: ["remap"]
    target: "stdout"
    encoding: "json"
VALUES
```
```

@jszwedko @binarylogic does this work as the quickstart config, or do we want to coalesce a number of parse functions?

Reply:
Should ` . = merge(., structured)` be ` . = merge(., object!(structured))`? I see:

```
  │
5 │                             . = merge(., structured)
  │                             --- ^^^^^^^^^^^^^^^^^^^^
  │                             │   │
  │                             │   this expression is fallible
  │                             │   update the expression to be infallible
  │                             or change this to an infallible assignment:
  │                             ., err = merge(., structured)
```

running this config. Alternatively, we could handle that error separately.

I think this config as an example makes sense though.

Reply:
Doh

Reply:
Looks good to me 👍